docker_configs/my_config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
please remove this file from your PR, it's a local config for your testing

Reply:
Hi Ellen, thank you so much for your reply. It's my first PR. After I read your reply, I realize it's not a  proper PR. I will fix my PR, close this one, and commit to your repo with another proper one. Thanks again.

Reply:
Thank you, Ellen. Thanks for your kind advice. 😊